mud_files/controller_bacnet_disabled.json

```diff
@@ -0,0 +1,148 @@
+{
+  "ietf-mud:mud": {
+    "mud-version": 1,
+    "mud-url": "https://digital-building.org/mud/controller",
+    "last-update": "2018-09-23T20:43:36+02:00",
+    "cache-validity": 48,
+    "is-supported": true,
+    "systeminfo": "Generic BACnet controller",
```

Comment:
Should name this what it *is* not what it isn't. So, this is some kind of system, that does X and Y -- what is that?  Eventually we'll want to use it for something else, so should be clear about that.  (Description and filename)

Reply:
Specifically -- how is this file different than the generic controller?

Reply:
Yes, very good point. I totally forgot to edit the system info description field, sorry!
I'll also change the names of the devices. 
I'm OK with the "positive" naming but not sure how to name the "bacnet disabled". Maybe in this instance I can omit the file altogether since default.json has no acls in it ...